components/net/fetch/methods.rs

```diff
@@ -109,15 +110,19 @@ pub async fn fetch(request: &mut Request, target: Target<'_>, context: &FetchCon
     fetch_with_cors_cache(request, &mut CorsCache::default(), target, context).await;
 }
 
-/// Continuation of fetch from step 9.
+/// Continuation of fetch from step 8.
 ///
 /// <https://fetch.spec.whatwg.org#concept-fetch>
 pub async fn fetch_with_cors_cache(
```

Comment:
I changed this and its calling function to take ownership of the Request so that I can instantiate the `FetchParams`. Is that ok? In the [original call to fetch](https://github.com/servo/servo/blob/bcad0d50e96cc9ffe190d0bbe2519df1bcadcfd3/components/net/resource_thread.rs#L769), the `Request` doesn't get used for anything else.

Reply:
Sure, `fetch_with_cors_cache` is the continuation of `fetch` iiuc, so it can take ownership of whatever it needs.